### PR TITLE
fix: Range Slider Mobile UX

### PR DESCRIPTION
**Pull request by dbchristopher**
* Restores range slider mobile support, which was reverted from previous release due to helltool integration bugs.
* Fixes issue where trackpad tap-to-click would get stuck in drag mode

https://looker.atlassian.net/browse/LENS-1399
https://looker.atlassian.net/browse/LENS-1431

range-slider-mobile
range-slider-desktop

### CODE REVIEW

**Comment by dbchristopher** [comment]
Tap-to-click was fixed in [this commit](https://github.com/looker-open-source/components/pull/1909/commits/cb4c9dcf55be22d71f6b6e8dfb68fb758334554d). There was some event ordering issues where the mouse events would fire so fast that it `mouseDown` would be the last thing to be fired

**Comment by dbchristopher** [comment]
@lukelooker @mdodgelooker I think this is safe to merge if you want to take another look! The easiest solution I found was literally just to copy the RangeSlide original source into filter components, and just run it directly there. If you want to try it out locally, you can check out pull request #59342 in our internal repo. 

I'm not seeing any evidence of the weird integration issues we saw earlier. So either it was an unrelated transient bug that was fixed in the past few months, or there is an issue with the transpilation and bundling. But I think it's safe to try releasing again 🙏 

Also includes a fix for the trackpad issue that Meredith discovered last time.

File: packages/components/CHANGELOG.md [hunk 1]
```diff
@@ -100,12 +100,6 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 - Clicking outside a focus trap deactivates it
 
-## [0.9.28]
-
-### Removed
-
-- Reverted: `RangeSlider` now supports touch events
-
```
Comment (lukelooker) [inline packages/components/CHANGELOG.md:108]: This should stay in place I guess?

**Review by lukelooker — APPROVED** [review]
LGTM :)

Let's make sure to test this specifically post-integration

**Comment by dbchristopher** [comment]
@lukelooker I restored the changelog note. Need another ✅

**Review by lukelooker — APPROVED** [review]

**Pull request merged**